src/commands/invest/index/processor.ts

```diff
@@ -35,7 +39,7 @@ function groupByToken(data: ApiEarningOption[]) {
         value: String(maxApy),
         fractionDigits: 2,
       }) + "%",
-    ]).join("~")
+    ]).join(" ~ ")
```

Comment:
use `APPROX`, it's a constant

Reply:
APPROX không đúng trong trong trường hợp này nên anh thêm `DASH` vào

Reply:
ah my bad, nó là min ~ max 👍